### DISCUSSION TITLE
RasPi-5 and 2 cameras

### DISCUSSION THREAD

**Discussion by Mattias-B** [bluenviron/mediamtx#3914] [Questions] [answered]
### Question

Since the Raspberry Pi 5 sports 2 camera connectors:
Is it possible in mediamtx, to address both raspi-cameras, say as "cam1" and "cam2", and have them both stream over rtsp at the same time?
What would the neccessary config file entries look like?

// Mattias

**Comment by beyonlo** [comment]
Remember that pi5 does not have the hardware video encoding feature like pi4, pi3 and pi 2w. So, to video encoding the pi5 use just software video encoding, using in this way more CPU usage. Of course Pi5 has more CPU power than olders Pi's.

Taking the moment, I would like to ask to you guys: does mediamtx use automatically hardware video encoding when used in pi4, pi3 and pi2w, and software video encoding in the pi5? Or is needed to extra config depending what pi is used?

Thank you!

**Reply by aler9** [reply]
the encoding topic is completely off topic here.

**Comment by aler9** [comment, marked as answer]
@Mattias-B hello, yes, you can stream two cameras at once, there's `rpiCameraCamID` for that:

```yml
paths:
  cam1:
    source: rpiCamera
    rpiCameraCamID: 0

  cam2:
    source: rpiCamera
    rpiCameraCamID: 1
```
